src/tests/ftest/pool/multi_server_create_delete_test.py

```diff
@@ -64,49 +59,63 @@ def test_create(self):
         host1 = self.hostlist_servers[0]
         host2 = self.hostlist_servers[1]
         test_destroy = True
-        data = dmg.pool_create(
-            "1GB", user, group, None, tgtlist, None)
-        if dmg.result.exit_status == 0:
+
+        self.add_pool(create=False)
+        self.pool.uid = user
+        self.pool.gid = group
+        self.pool.target_list.update(tgtlist)
+
+        # Disable raising an exception if the dmg command fails
+        self.pool.dmg.exit_status_exception = False
+
+        self.pool.create()
```

Comment:
Instead of ensuring the `self.pool.dmg.exit_status_exception` is would probably be better to ensure it is reset in `tearDown()` in order to catch all the cases where it may not have been properly reset.  In other words this should be handled in another PR and can land as it is today.